documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -226,7 +226,7 @@ make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- bcm2711_defconfig
 
 ===== Build with Configs
 
-NOTE: To speed up compilation on multiprocessor systems, and get some improvement on single processor ones, use `-j n`, where n is the number of processors * 1.5. Alternatively, feel free to experiment and see what works!
+NOTE: To speed up compilation on multiprocessor systems, and get some improvement on single processor ones, use `-j n`, where n is the `nproc`(number of processors) * 1.5. Alternatively, feel free to experiment and see what works!
```

Comment:
I wonder if it might be better as "... where _n_ is the number of processors * 1.5. You can use the `nproc` command to see how many processors you have. Alternatively, feel free to experiment and see what works!"
:shrug:  But of course @aallan gets to make the final decision :slightly_smiling_face:

Reply:
@lurch I reflected it. Please check it.